src/passes/PrintCallGraph.cpp

```diff
@@ -34,72 +35,133 @@ struct PrintCallGraph : public Pass {
   bool modifiesBinaryenIR() override { return false; }
 
   void run(Module* module) override {
-    std::ostream& o = std::cout;
-    o << "digraph call {\n"
-         "  rankdir = LR;\n"
-         "  subgraph cluster_key {\n"
-         "    node [shape=box, fontname=courier, fontsize=10];\n"
-         "    edge [fontname=courier, fontsize=10];\n"
-         "    label = \"Key\";\n"
-         "    \"Import\" [style=\"filled\", fillcolor=\"turquoise\"];\n"
-         "    \"Export\" [style=\"filled\", fillcolor=\"gray\"];\n"
-         "    \"Indirect Target\" [style=\"filled, rounded\", "
-         "fillcolor=\"white\"];\n"
-         "    \"A\" -> \"B\" [style=\"filled, rounded\", label = \"Direct "
-         "Call\"];\n"
-         "  }\n\n"
-         "  node [shape=box, fontname=courier, fontsize=10];\n";
-
-    // Defined functions
-    ModuleUtils::iterDefinedFunctions(*module, [&](Function* curr) {
-      std::cout << "  \"" << curr->name
-                << "\" [style=\"filled\", fillcolor=\"white\"];\n";
-    });
-
-    // Imported functions
-    ModuleUtils::iterImportedFunctions(*module, [&](Function* curr) {
-      o << "  \"" << curr->name
-        << "\" [style=\"filled\", fillcolor=\"turquoise\"];\n";
-    });
-
-    // Exports
-    for (auto& curr : module->exports) {
-      if (curr->kind == ExternalKind::Function) {
-        Function* func = module->getFunction(curr->value);
-        o << "  \"" << func->name
-          << "\" [style=\"filled\", fillcolor=\"gray\"];\n";
-      }
-    }
+    Name caller = getPassOptions().getArgumentOrDefault(
+      "func",
```

Comment:
As a convention we use the pass in the name of the flag (unless the flag applies to more than one pass somehow). See for example the `--jspi` pass which uses `jspi-*` names: https://github.com/WebAssembly/binaryen/blob/6a429411a07a35ec31af6a66d04dbf4637bcf2ef/src/passes/JSPI.cpp#L88-L97

Reply:
Resolved in https://github.com/WebAssembly/binaryen/pull/6354/commits/10f8643e345aa8176ecbe97b00e8c62c0dc09d73